disasm/disasm.cc

```diff
@@ -855,7 +855,7 @@ disassembler_t::disassembler_t(int xlen)
 
   DEFINE_NOARG(ecall);
   DEFINE_NOARG(ebreak);
-  DEFINE_NOARG(uret);
+//  DEFINE_NOARG(uret);
```

Comment:
We shouldn't have commented-out code.  (I'm OK with removing `uret` as a separate PR, though.)